internal/usage/usage.go

```diff
@@ -249,9 +249,9 @@ dbName and collection are required only for built-in roles.`
 	LinkTokenAccessListCIDREntries            = "IP address access list entries that are associated with the link-token." //nolint:gosec // This is just a message not a credential
 	LinkToken                                 = "Link-token generated by Atlas."                                          //nolint:gosec // This is just a message not a credential
 	LiveMigrationID                           = "Unique 24-hexadecimal digit string that identifies the live migration job."
-	PrivateEndpointID                         = "Unique identifier of the AWS PrivateLink connection."
-	EndpointServiceID                         = "Unique identifier of the private endpoint service for which you want to retrieve a private endpoint."
-	PrivateEndpointIDAzure                    = "Unique identifier of the Azure private endpoint resource."
+	PrivateEndpointID                         = "Unique 22-character alphanumeric string that identifies the AWS PrivateLink connection in AWS. You can find this value on the AWS VPC Dashboard under Endpoints > VPC ID."
+	EndpointServiceID                         = "Unique 24-character alphanumeric string that identifies the private endpoint in Atlas."
+	PrivateEndpointIDAzure                    = "Unique string that identifies the Azure private endpoint in Azure. The Properties page for your private endpoint on your Azure dashboard displays this property in the Resource ID field."
```

Comment:
Nit: Since the AWS one specifies the length, I think this one should too
```suggestion
	PrivateEndpointIDAzure                    = "Unique 24-character alphanumeric string that identifies the Azure private endpoint in Azure. The Properties page for your private endpoint on your Azure dashboard displays this property in the Resource ID field."
```

Reply:
The length of the Azure one varies based on endpoint name and other things (it looks like this: /subscriptions/4e133d35-e734-4385-a565-c0945567ae346/resourceGroups/rg_95847a959b876e255dbb9b33_dfragd7w/ providers/Microsoft.Network/privateEndpoints/test-endpoint) so I will leave this one as-is